src/OpenTelemetry.Exporter.Jaeger/JaegerExporterProtocolParser.cs

```diff
@@ -16,13 +16,21 @@
 
 namespace OpenTelemetry.Exporter;
 
-internal static class JaegerExporterOptionsExtensions
+internal static class JaegerExporterProtocolParser
 {
-    public static JaegerExportProtocol? ToJaegerExportProtocol(this string protocol) =>
-        protocol?.Trim() switch
+    public static bool TryParseProtocol(string value, out JaegerExportProtocol result)
```

Comment:
The class is already named `JaegerExporterProtocolParser` then I think we can simply rename the method to `TryParse`